app/assets/javascripts/notification.js.coffee

```diff
@@ -0,0 +1,11 @@
+$(document).on 'click', '[data-behavior~=update-is_read]', (_this) ->
+  $.ajax
+    type: 'PUT'
+    url: '/notifications/' + @id
+    data:
+      is_read: @checked
+      id: @id
+  if @checked
+    document.getElementById('notification' + @id).classList.add('read_notification')
```

Comment:
You can reduce duplication by introducing a variable and only have something like `yourVariable.add(...) and `remove` in condition branches.
(Try <kbd>Ctrl+Alt+V</kbd> on a selected expression in RubyMine to introduce a variable.)